src/main/java/tech/fastj/engine/FastJEngine.java

```diff
@@ -702,7 +702,7 @@ private static void sync() {
             try {
                 TimeUnit.MILLISECONDS.sleep((long) ((endTime - currentTime) * 1000L));
             } catch (InterruptedException exception) {
-                FastJEngine.warning("Interrupted while syncing game frame rate: " + exception.getMessage() + Arrays.deepToString(exception.getStackTrace()));
+                FastJEngine.warning("Interrupted while syncing game frame rate: {}" + exception.getMessage() + Arrays.deepToString(exception.getStackTrace()));
```

Comment:
Seems like you've added the {} formatting, but the string concatenation is still present.